packages/types/src/documents/app/view.ts

```diff
@@ -79,10 +79,15 @@ export enum CalculationType {
   MAX = "max",
 }
 
+export enum ViewV2Type {
+  CALCULATION = "calculation",
```

Comment:
Should we have a name for the 'non-calculation' views, and to start populating them as well?

Reply:
I thought this too, but unless we run a migration to backfill it then the inconsistency would annoy me. I don't mind either way though.

Reply:
I would argue no, it's a bit less messy to assume if type is not there, it's a "vanilla" ViewV2. If we start populating it, we will infinitely have to check for either not present, or equal to some value. I'd prefer one state to two.